### PR TITLE
mail issues

### PR DESCRIPTION
**Pull request by ksiedlarek**

### CODE REVIEW

File: jobs/admin.py [hunk 1]
```diff
@@ -21,82 +19,31 @@ def make_published(modeladmin, request, queryset):
...
-        send_from = "jobs@djangogirls.org"
-        recipient = [item.contact_email, ]
-        send_mail(
+        recipient = item.contact_email
+        status = send_job_mail(
             subject,
             message_plain,
-            send_from,
+            message_html,
             recipient,
```
Comment (rebkwok) [inline jobs/admin.py:39]: Does recipient not need to be a list?

Comment (rebkwok) [inline jobs/admin.py:39]: Ah, nevermind, I see where it's actually sent now :)

File: core/migrations/0011_auto_20150707_1518.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            model_name='postmortem',
+            name='feedback',
+            field=models.TextField(blank=True, verbose_name='How we can make DjangoGirls better?', null=True),
+        ),
+        migrations.AlterField(
+            model_name='story',
+            name='image',
+            field=models.ImageField(upload_to='stories/'),
+        ),
+    ]
```
Comment (rebkwok) [inline core/migrations/0011_auto_20150707_1518.py]: I don't think we should have needed to make additional migrations for applications and core - they should have been made in the main repo and just needed migrate run once they'd been pulled in.  Can you see if you are able to get a new clone of the master djangogirlsjobs repo running without doing makemigrations on these two apps?  We're likely to get conflicts with the main repo if we make migrations in these?

File: jobs/migrations/0004_auto_20150707_1518.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            model_name='meetup',
+            name='review_status',
+            field=models.CharField(choices=[('OPN', 'Open'), ('URE', 'Under review'), ('RTP', 'Ready to publish'), ('REJ', 'Rejected'), ('PUB', 'Published')], max_length=3, default='OPN'),
+        ),
+        migrations.AlterField(
+            model_name='meetup',
+            name='website',
+            field=models.URLField(blank=True, null=True, help_text='Link to your meetup or organisation website.'),
+        ),
+    ]
```
Comment (MarysiaLowas) [inline jobs/migrations/0004_auto_20150707_1518.py:64]: What is in this migration? I cannot work it out for myself ;) I'm asking as I'm preparing another PR with one migration and I would to know what is changing.

File: core/migrations/0011_auto_20150707_1518.py (hunk 2 above)
Comment (ksiedlarek) [inline core/migrations/0011_auto_20150707_1518.py]: I can remove it if it's needed, as it is the last migration from what I see. I didn't change anything in there. I also cloned our repo and did only syncdb (and it didn't create new migrations for core app). What should I do?

File: jobs/migrations/0004_auto_20150707_1518.py (hunk 3 above)
Comment (ksiedlarek) [inline jobs/migrations/0004_auto_20150707_1518.py:64]: I've added:
 from **future** import unicode_literals and **str** method
changed those bits with sending email and formatting and I think that's all

File: core/migrations/0011_auto_20150707_1518.py (hunk 2 above)
Comment (rebkwok) [inline core/migrations/0011_auto_20150707_1518.py]: Yes, remove the core and applications migrations, and if you need to make a migration on the jobs app, do it only there, with `./manage.py makemigrations jobs`.  I'm guessing it created these two when you ran makemigrations for the jobs migration you had in this PR.

**Pull request merged**